modules/gdnative/SCsub

```diff
@@ -12,6 +12,8 @@ gdn_env.add_source_files(env.modules_sources, "nativescript/*.cpp")
 
 gdn_env.Append(CPPPATH=['#modules/gdnative/include/'])
 
+SConscript("nativearvr/SCsub")
```

Comment:
We should likely define how gdnative submodules are meant to be built and stick to it. This one uses its own SCsub, nativescript has its own SCsub but it's not used and instead the files are included above, and pluginscript uses the same system as nativescript, albeit without the (unused) SCsub.

Reply:
@karroffel what is your take on this? We briefly discussed this when I love this into the gdnative folder. I personally feel each folder should have its own SCSub as its easier to find but I'm also up for removing it. 

@akien-mga nativescript has its own SCSub because just like nativearvr it started life on its own it seems, we should either change nativearvr to just using the main SCSub and delete both files from the subfolders, or start using the SCsub in nativescript :)

Reply:
@akien-mga I just forgot that this feature existed when I extracted nativescript, I advised him to use SConscript because I think that's the better way and I just did it ugly.

Reply:
Ok, so @touilleMan should likely do it the same way for pluginscript, and we should fix it for nativescript to be consistent.

Reply:
Yes